test/ttmlir/Dialect/TTNN/eltwise/unary/sign/simple_sign.mlir

```diff
@@ -0,0 +1,12 @@
+// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline %s | FileCheck %s
+#any_device = #tt.operand_constraint<dram|l1|scalar|tile|any_device|any_device_tile>
+module attributes {} {
+  func.func @forward(%arg0: tensor<64x128xf32>) -> tensor<64x128xf32> {
+    %0 = tensor.empty() : tensor<64x128xf32>
+    // CHECK: [[VAL0:%[0-9]+]] = "ttnn.empty"(%{{[0-9]+}}) <{dtype = {{.*}}, layout = {{.*}}, memory_config = {{.*}}, <{{.*}}>>, shape = #ttnn.shape<[[TENSOR_SHAPE:[0-9]+x[0-9]+]]>}>
+    %1 = "ttir.sign"(%arg0, %0) <{operandSegmentSizes = array<i32: 1, 1>, operand_constraints = [#any_device, #any_device]}> : (tensor<64x128xf32>, tensor<64x128xf32>) -> tensor<64x128xf32>
+    // CHECK: %{{[0-9]+}} = "ttnn.sign"(%{{[0-9]+}}, [[VAL0]]) <{operandSegmentSizes = array<i32: 1, 1>}> : (tensor<[[TENSOR_SHAPE]]x{{.*}}, {{.*}}>, tensor<[[TENSOR_SHAPE]]x{{.*}}, {{.*}}) -> tensor<[[TENSOR_SHAPE]]x{{.*}}, {{.*}}>
```

Comment:
You can capture this into VAL1 to use with return check

Reply:
As the return argument is different from the result of the sign operation (due to the "ttnn.to_memory_config" operation in between), I did not capture the result to be used with the return check. Example of translated code is below.
```
    %4 = "ttnn.sign"(%2, %3) <{operandSegmentSizes = array<i32: 1, 1>}> : (tensor<64x128xf32, #layout1>, tensor<64x128xf32, #layout1>) -> tensor<64x128xf32, #layout1>
    "ttnn.dealloc"(%2) : (tensor<64x128xf32, #layout1>) -> ()
    %5 = "ttnn.to_memory_config"(%4, %0) : (tensor<64x128xf32, #layout1>, !tt.device<#device>) -> tensor<64x128xf32, #layout>
    "ttnn.dealloc"(%3) : (tensor<64x128xf32, #layout1>) -> ()
    return %5 : tensor<64x128xf32, #layout>
```

Reply:
As the return argument is different from the result of the sign operation (due to the "ttnn.to_memory_config" operation in between), I did not capture the result to be used with the return check. Example of translated code is below.
```
    %4 = "ttnn.sign"(%2, %3) <{operandSegmentSizes = array<i32: 1, 1>}> : (tensor<64x128xf32, #layout1>, tensor<64x128xf32, #layout1>) -> tensor<64x128xf32, #layout1>
    "ttnn.dealloc"(%2) : (tensor<64x128xf32, #layout1>) -> ()
    %5 = "ttnn.to_memory_config"(%4, %0) : (tensor<64x128xf32, #layout1>, !tt.device<#device>) -> tensor<64x128xf32, #layout>
    "ttnn.dealloc"(%3) : (tensor<64x128xf32, #layout1>) -> ()
    return %5 : tensor<64x128xf32, #layout>
```

Reply:
Oh right, I forgot about that. All good then.